api/client/samples/crop_models/Sao_Paulo_brazil_sugar_content_modeling.ipynb

```diff
@@ -0,0 +1,3629 @@
+{
```

Comment:
<p>Why is the client wrapper set twice here?</p><p> </p><p>model = CropModel('api.gro-intelligence.com', os.environ['GROAPI_TOKEN'])</p><p> </p><p>This has already been done at the start of the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>